uint8 UP = 1
uint8 DOWN = 2
uint8 MID = 3

float64 ch_l_x
float64 ch_l_y
float64 ch_r_x
float64 ch_r_y
#switch
uint8 s_l
uint8 s_r
float64 wheel
#mouse
float64 m_x
float64 m_y
float64 m_z
bool p_l
bool p_r
#key board
bool[16] key
time stamp